x-pack/plugins/security_solution/public/onboarding/components/onboarding_body/cards/common/card_content_asset_panel.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useMemo, type PropsWithChildren } from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiSpacer } from '@elastic/eui';
+import { OnboardingCardContentPanel } from './card_content_panel';
+import { useCardContentAssetPanelStyles } from './card_content_asset_panel.styles';
+import { CardSelectorListItemAssetType } from '../types';
+
+interface OnboardingCardContentAssetType {
+  type: CardSelectorListItemAssetType;
+  source: string;
+  alt: string;
+}
+
+export const OnboardingCardContentAssetPanel = React.memo<
+  PropsWithChildren<{
+    asset: OnboardingCardContentAssetType;
+  }>
+>(({ children, asset: { type, source, alt } }) => {
```

Comment:
This type is already defined in _types.ts_, let's not duplicate it:

```suggestion
type CardContentAssetPanelProps =   PropsWithChildren<{ asset: CardAsset; }>

export const OnboardingCardContentAssetPanel = React.memo<
  CardContentAssetPanelProps
>(({ children, asset: { type, source, alt } }) => {
```